test/sets.jl

```diff
@@ -1,5 +1,9 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
+# OffsetVector is used to test `unique!` on non-standard indices (PR #20619).
+isdefined(Main, :TestHelpers) || (@eval Main include(joinpath(dirname(@__FILE__),
+   "TestHelpers.jl")))
```

Comment:
Line must be aligned with `dirname` since that's an argument to the same function. Though here probably better break before `(@eval`, or even better use an `if isdefined...` multiline block.